File: Core/Entities/Location.cs
using NetTopologySuite.Geometries;

namespace Core.Entities
{
    public class Location : BaseEntity
    {
        public string Street { get; set; }
        public string City { get; set; }
        public string Description { get; set; }
        public Point Located { get; set; }
        public string Picture { get; set; }

        public int CountryId { get; set; }
        public Country Country { get; set; }
    }
}

File: client/src/app/shared/models/location.ts
export interface Location1 {
    id: number;
    street: string;
    city: string;
    description: string;
    latitude: number;
    longitude: number;
    picture: string;
    workingHours: string;
    email: string;
    phone: string;
    country: string;
}

export interface LocationCreateEdit {
    id: number;
    street: string;
    city: string;
    description: string;
    latitude: number;
    longitude: number;
    picture: string;
    workingHours: string;
    email: string;
    phone: string;
    countryId: number;
}

export class LocationCreateEdit {
    id: number;
    street: string;
    city: string;
    description: string;
    latitude: number;
    longitude: number;
    picture: string;
    workingHours: string;
    email: string;
    phone: string;
    countryId: number;
}



